tests/Doctrine/Tests/ODM/PHPCR/Functional/Hierarchy/ChildTest.php

```diff
@@ -471,7 +471,7 @@ class ChildTestObj
   */
 class ChildReferrerTestObj
 {
-  /** @PHPCRODM\Id */
+    /** @PHPCRODM\Id */
```

Comment:
sure .. but it doesn't fix everything:
https://github.com/doctrine/phpcr-odm/blob/styleci/tests/Doctrine/Tests/ODM/PHPCR/Functional/Hierarchy/ChildTest.php#L472